src/subtype.c

```diff
@@ -1663,6 +1679,8 @@ JL_DLLEXPORT int jl_obvious_subtype(jl_value_t *x, jl_value_t *y, int *subtype)
             if (i < nparams_expanded_x) {
                 // there are elements left in x (possibly just a Vararg), check them against the Vararg tail of y too
                 assert(vy != JL_VARARG_NONE && istuple && iscov);
+                if (vararg_elt_bound(jl_tparam(y, i)))
+                    return 0;
```

Comment:
Isn’t this just a rather complicated way of recomputing vt and then always returning false from here, or am I forgetting how normalization works now?

Reply:
Ah, you're right. I guess instead we should check whether T in `Vararg{T}` is a typevar and whether it occurs invariant?

Reply:
yeah, I guess we need to store a reference to the outermost type object so we could walk that here and look for uses inside invariant contexts (disabling some of the exactness tests below, but keeping the subtyping ones)?